### ISSUE TITLE
Allow comments on study acceptance as well as rejection

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#610]
# TL;DR
Allow reviewer to leave comments on study acceptance as well as rejection

# Narrative
As a study reviewer, I often want to leave final comments along with study acceptance, either to offer softer advice (no changes required, but hey this might come up in piloting or you might consider doing X in the future) or just to say something was nicely done. Currently this has to be done via email/Slack, and any advice does not end up part of the study log.

# Acceptance Criteria

- [ ] Comments can be added with study acceptance
- [ ] Comments still can't be added with pausing/starting/etc., just accept/reject/submit. (It actually wouldn't be unreasonable to consider allowing comments on almost everything to support more of a lab notebook - "we paused because we reached N kids and are going to code now to see how many more we need per condition," etc. - but for now let's just make the minimal change.)
- [ ] Comments are added to the approval decision email
- [ ] Appropriate tests added: this will depend on whether we're refactoring the StudyDetailView first per #518 , see below.

# Implementation notes

- The "update status" request is handled by the exp/views/study.py/StudyDetailView view. This currently handles three unrelated form submissions, which makes very little sense - there should in principle be three separate views for adding/removing users, changing study status, and cloning the study. See https://github.com/lookit/lookit-api/issues/518.  Refactoring now could be a good exercise if you'd prefer to do it first (in which case we'll take the other refactor off the onboarding list) but we could also hold off, we're not making anything worse :)
- Whether to show the comments box is just decided in Javascript in studies/templates/studies/study_detail.html/onStateSelect, and then in exp/views/study.py/update_trigger we check if comments-text was sent with the request. So we probably only need to change this in the template to allow approval comments.
- See studies/models.py/Study/notify_submitter_of_approval for function that sends email about approval (will need to modify that email template to include comments, like for rejection emails)
- Tests: If refactoring the StudyDetailView, now is the time to add basic tests of each post view - adding/updating user perms, changing study state, & cloning study. Otherwise, would tentatively propose adding:
   - Test that no study views are accessible if not a researcher (e.g., test_cannot_see_any_study_views_as_participant analogous to exp/tests/test_study_views.py/test_cannot_see_any_study_views_unauthenticated) 
   - Test that *getting* StudyDetailView requires appropriate permissions (i.e., StudyPermission.READ_STUDY_DETAILS or LabPermission.READ_STUDY_DETAILS for the study's associated lab)
